packages/@uppy/audio/types/index.d.ts

```diff
@@ -0,0 +1,12 @@
+import type { PluginOptions, UIPlugin, PluginTarget } from '@uppy/core'
+import AudioLocale from './generatedLocale'
+
+export interface AudioOptions extends PluginOptions {
+    target?: PluginTarget
+    showVideoSourceDropdown?: boolean
+    locale?: AudioLocale
+}
+
+declare class Audio extends UIPlugin<AudioOptions> {}
```

Comment:
any methods that should be publicly available through `.getPlugin`? If so, then they need to be typed here (or all made #private in the code itself)